net/lora-packet-forwarder/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2019 Xue Liu <liuxuenetmail@gmail>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lora-packet-forwarder
+PKG_VERSION:=4.0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/Lora-net/packet_forwarder/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=e68fadf6f1d2e5e7b601e504d5efb48b0a8f374c2c29c0476ab2fe9db68d33ae
+PKG_MAINTAINER:=Xue Liu <liuxuenetmail@gmail.com>
+PKG_LICENSE_FILES:=LICENSE
```

Comment:
What's the license?

Reply:
https://github.com/Lora-net/packet_forwarder/blob/master/LICENSE

They came up with their own.

Reply:
Looks like BSD to me.

Reply:
So PKG_LICENSE:=BSD-2-Clause?

Reply:
I see three clauses. Unless I'm mistaken.